template.yml

```diff
@@ -111,6 +115,33 @@ objects:
     username: ${AWX_USER}
     password: ${AWX_PASSWORD}
 
+- apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Nit: you should be able to use the `system:auth-delegator` ClusterRole which already exists in OpenShift/Kubernetes and is equivalent to the role created here.